pom.xml

```diff
@@ -146,34 +200,24 @@
                 <version>${amps.version}</version>
                 <extensions>true</extensions>
                 <configuration>
+                    <banningExcludes>
```

Comment:
the banning rules prevent importing guava and gson because "the platform provides them" - but they have been removed in Platform 7, so I believe this ban is wrong. Anyhow, I found another plugin on dotcom that does a similar thing so seems fine to me.